### DISCUSSION TITLE
纪念一个游戏服务器

### DISCUSSION THREAD

**Discussion by CnData07** [share121/inter-knot#2075] [其他]
![77a7e332e55a565d02213ae20177b1a1_720](https://github.com/user-attachments/assets/79b9251f-7d0f-49c1-8e01-2f461496af24)
被乐子人网暴退圈了，开服到现在每个月定亏800，开了一年，开不下去了，再见我的青春，以后真不想玩mc了。

**Comment by poolofwater** [comment]
抱抱，愿你以后依旧能在mc找到快乐<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >
